index.html

```diff
@@ -60,5 +61,387 @@ <h2>I am a Software Developer</h2>
+                    />
+                  </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <img
+                      alt="rubyButton"
+                      src="img/rubyTransparent.png"
+                      class="buttonPropertiesWorks"
+                    />
+                  </a>
+                </li>
+              </ul>
+              <a href="#">
+                <img
+                  alt="seeProjectButton"
+                  src="img/Action.png"
+                  class="buttonProperties"
+                />
+              </a>
+            </div>
+            <div class="gridItem grid8 backGroundImg">
+              <h2>
+                Profesional Art<br />
+                Printing Data
+              </h2>
+              <p>
+                A daily selection of privately<br />
+                personalized reads; no accounts or<br />
+                sign-ups required. has been the<br />
+                industry's standard
+              </p>
+              <ul class="d-flex">
+                <li>
+                  <a href="#">
+                    <img
+                      alt="htmlButton"
+                      src="img/htmlTransparent.png"
+                      class="buttonPropertiesWorks"
+                    />
+                  </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <img
+                      alt="bootstrapButton"
+                      src="img/bootstrapTransparent.png"
+                      class="buttonPropertiesWorks"
+                    />
+                  </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <img
+                      alt="rubyButton"
+                      src="img/rubyTransparent.png"
+                      class="buttonPropertiesWorks"
+                    />
+                  </a>
+                </li>
+              </ul>
+              <a href="#">
+                <img
+                  alt="seeProjectButton"
+                  src="img/Action.png"
+                  class="buttonProperties"
+                />
+              </a>
+            </div>
+          </div>
+        </section>
+        <section class="aboutMe">
+          <div class="d-flex bottom flex-center">
+            <h2>About me</h2>
+            <p>
+              Hello I’m a software developer! I can help<br />
+              you build a product, feature or website<br />
+              Look through some of my work and<br />
+              experience! If you like what you see and<br />
+              have a project you need coded, don’t<br />
+              hestiate to contact me.
+            </p>
+            <a href="#">
+              <img alt="resume" src="img/Resume.png" />
+            </a>
+          </div>
+          <img alt="division" src="img/line2.png" class="line2" />
+        </section>
```

Comment:
- [ ] Please notice in this About me section you have a lot of text that isn't using the correct font. please ensure that you are using the font specified on the Figma reference design.

|Your Design | Reference Design |
-|-
|![image](https://user-images.githubusercontent.com/18730106/153726831-8833568e-da2c-4ca7-9e03-e726e925c9df.png)|![image](https://user-images.githubusercontent.com/18730106/153726834-e0649e6b-6f70-40f2-8fda-fcca534fe1ed.png)|
|![image](https://user-images.githubusercontent.com/18730106/153726847-2d586ea0-3851-4a02-a88c-704774edb8b8.png)|![image](https://user-images.githubusercontent.com/18730106/153726851-d6930bf6-f3c8-4f24-a60d-b89fbbef4f5b.png)|